test/shared/src/main/scala/zio/test/Spec.scala

```diff
@@ -112,17 +112,20 @@ final case class Spec[-R, +E](caseValue: SpecCase[R, E, Spec[R, E]]) extends Spe
    * spec. If no labels satisfy the specified predicate then returns `Some` with
    * an empty suite if this is a suite or `None` otherwise.
    */
-  final def filterLabels(f: String => Boolean)(implicit trace: Trace): Option[Spec[R, E]] =
+  final def filterLabels(f: String => Boolean, prefix: String = "", accumulatePrefix: Boolean = true)(implicit
```

Comment:
This isn't binary compatible (which is why CI is failing). You'll have to introduce a new method called `filterLabels` with the additional arguments, and proxy to it from the original method. i.e.,:

```scala
final def filterLabels(f: String => Boolean)(implicit trace: Trace): Option[Spec[R, E]] = 
  filterLabels(f, "")

final def filterLabels(f: String => Boolean, prefix: String, accumulatePrefix: Boolean = true)(implicit trace: Trace): Option[Spec[R, E]] = {
  // implementation goes here
}

Reply:
> You'll have to introduce a new method ...

Thank you for the clue!!
Done.